android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,297 @@
+
+    }
+
+    @Override
+    public void onLowOrRaiseHandStatusChanged(long l, boolean b) {
+
+    }
+
+    @Override
+    public void onMeetingSecureKeyNotification(byte[] bytes) {
+
+    }
+
+    @Override
+    public void onChatMessageReceived(InMeetingChatMessage inMeetingChatMessage) {
+
+    }
+
+    @Override
+    public void onSilentModeChanged(boolean b) {
+
+    }
+
+    @Override
+    public void onFreeMeetingReminder(boolean b, boolean b1, boolean b2) {
+
+    }
+
+    @Override
+    public void onMeetingActiveVideo(long l) {
+    }
+
+    @Override
+    public void onSinkAttendeeChatPriviledgeChanged(int i) {
+
+    }
+
+    @Override
+    public void onSinkAllowAttendeeChatNotification(int i) {
+
+    }
+
+    @Override
+    public void onUserNameChanged(long l, String s) {
+
+    }
+
+//    @Override
+//    public void onUserVideoStatusChanged(long l, VideoStatus videoStatus) {
+//        WritableMap params = extractUserInfoFromUserId(l);
+//        String status;
+//        switch (videoStatus) {
+//            case Video_ON:
+//                status = "on";
+//                break;
+//            case Video_OFF:
+//                status = "off";
+//                break;
+//            case Video_Mute_ByHost:
+//                status = "muteByHost";
+//                break;
+//            default:
+//                status = "undefined";
+//        }
+//        params.putString("status", status);
+//        notifyEvent("meeting.user.video.status", params);
+//    }
+
+    @Override
+    public void onUserVideoStatusChanged(long l) {
+        InMeetingUserInfo userInfo = inMeetingService.getUserInfoById(l);
+        WritableMap map = Arguments.createMap();
+        map.putString("name", userInfo.getUserName());
+        map.putString("userId", "" + userInfo.getUserId());
+        map.putBoolean("active", userInfo.getVideoStatus().isSending());
+        notifyEvent("meeting.user.video.status", map);
+    }
+//
+//    @Override
+//    public void onUserAudioStatusChanged(long l, AudioStatus audioStatus) {
+//        WritableMap params = extractUserInfoFromUserId(l);
+//        String status;
+//        switch (audioStatus) {
+//            case Audio_None:
+//                status = "none";
+//                break;
+//            case Audio_Muted:
+//                status = "muted";
+//                break;
+//            case Audio_Muted_ByHost:
+//                status = "muteByHost";
+//                break;
+//            case Audio_UnMuted:
+//                status = "unmute";
+//                break;
+//            case Audio_UnMuted_ByHost:
+//                status = "unmuteByHost";
+//                break;
+//            case Audio_MutedAll_ByHost:
+//                status = "mutedAllByHost";
+//                break;
+//            case Audio_UnMutedAll_ByHost:
+//                status = "unmutedAllByHost";
+//                break;
+//            default:
+//                status = "undefined";
```

Comment:
1. In lib `audioStatus` is ignored
2. In lib this event is triggered only for own user